app/src/main/java/cm/aptoide/pt/home/HomeAnalytics.java

```diff
@@ -27,6 +27,10 @@
   static final String TAP_ON_CARD_DISMISS = "tap on card dismiss";
   static final String TAP = "tap";
   static final String VIEW_CARD = "view card";
+  private static final String ACTION = "action";
+  private static final String BUNDLE_TAG = "bundle_tag";
+  private static final String PROMOTION_ICON = "promotion-icon";
```

Comment:
On the ticket it was written "promotions-icon". I know you just extracted the variable that was being used before. This might not be a problem, if Ana correctly analyses it on her side.

Reply:
Changed to promotion-icon